server/channels/app/mention_results.go

```diff
@@ -85,3 +81,11 @@ func (m *MentionResults) addMention(userID string, mentionType MentionType) {
 func (m *MentionResults) removeMention(userID string) {
 	delete(m.Mentions, userID)
 }
+
+func (m *MentionResults) addGroupMention(groupID string) {
+	if m.GroupMentions == nil {
+		m.GroupMentions = make(map[string]MentionType)
```

Comment:
It is good to give a capacity to the maps when creating them, but I guess we don't have a sensible size on this context.

Reply:
Yeah, I'd usually do that, but we don't have any way of knowing that at this point